L1Trigger/L1TMuon/interface/MuonRawDigiTranslator.h

```diff
@@ -59,6 +59,16 @@ namespace l1t {
     static constexpr unsigned etaMu1SignShift_ = 21;  // For Run-3
     static constexpr unsigned absEtaMu2Shift_ = 22;   // For Run-3
     static constexpr unsigned etaMu2SignShift_ = 30;  // For Run-3
+    static constexpr int ugmtFedId_ = 1402;
+    static constexpr int ugtFedId_ = 1404;
+    static constexpr int ugmtFwVersionUntil2016_ = 0x4010000;
+    static constexpr int ugtFwVersionUntil2016_ = 0x10A6;
+    static constexpr int ugmtFwVersionUntil2017_ = 0x6000000;
+    static constexpr int ugtFwVersionUntil2017_ = 0x1120;
+    static constexpr int ugmtFwVersionRun3Start_ = 0x6000001;
+    static constexpr int ugtFwVersionUntilRun3Start_ = 0x1130;
+    static constexpr int ugmtFwVersionFirstWithShowers_ = 0x7000000;
+    static constexpr int ugtFwVersionFirstWithShowers_ = 0x113B;
```

Comment:
Perhaps someone can correct me on this, but I think it is preferred that if you are adding class constants, that they have prefix `k` in front of them?

Reply:
Hi Andrew,

Ah, didn't know that but I'm happy to change it. Should I then do so for the other constants in that file too, I suppose? Also, is it fine to only do it in this PR, but not for the backport?

Cheers,
Dinyar

Reply:
@dinyar It should be fine to keep old constants how they are for now, but I believe that any newly added constants should have prefix `k` according to http://cms-sw.github.io/cms_coding_rules.html

Reply:
@dinyar And if possible I think it would be good to have the change (same commit if it all possible) in both this and the backport.

Reply:
Hi @aloeliger,

Ok, done now, let me know if it's ok. I've created a separate commit for the change, but if it also goes in the backport I'd be tempted to squash it (and do the same for the backport). Is that fine?

-d

Reply:
@dinyar I would think it's fine. The alternative is you could just `git cherry-pick` the same commit to the backport branch of your repository.

Reply:
Ok, done.